setup.py

```diff
@@ -54,7 +54,11 @@
         'License :: OSI Approved :: GNU Affero General Public License v3',
         'Operating System :: OS Independent',
         'Programming Language :: JavaScript',
-        'Programming Language :: Python',
+        'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
+        'Framework :: Django :: 2.0',
```

Comment:
1.11?

Reply:
In [BOM-1009](https://openedx.atlassian.net/browse/BOM-1009), it was mentioned to only add Django20, 21 & 22 environments so I removed django111 since we are dropping support for Django1.11 now.

Reply:
Again, not critical to add since we've already deployed this code to prod and are unlikely to make major changes until we're on 2.2, but it's always good to be testing the version we currently use in production.

Reply:
Added Django111 environment too.